web/src/components/molecules/Asset/UploadModal/uploadModal.tsx

```diff
@@ -20,6 +22,8 @@ type Props = {
 };
 
 const UploadModal: React.FC<Props> = ({
+  uploadButtonText,
+  cancelButtonText,
```

Comment:
Why are these needed? Couldn't find where they are being used.

Reply:
I removed unused arguments.

Reply:
Instead of passing the `uploadButtonText` string, can you pass `alsoLink` (boolean)? 
I feel like passing uploadButtonText works, but its not really specific enough I feel. If the UploadModal component was more generic, being able to change the button text might make sense. But this is pretty specifically for uploading, or uploading and linking, assets.

Reply:
Done!